cohere/client.py

```diff
@@ -226,24 +220,9 @@ def chat(
         if chatlog_override is not None:
             self._validate_chatlog_override(chatlog_override)
 
-        if session_id != "":
-            conversation_id = session_id
-            logger.warning(
-                "The 'session_id' parameter is deprecated and will be removed in a future version of this function. Use 'conversation_id' instead.",
-            )
-        if persona_prompt is not None:
-            preamble_override = persona_prompt
-            logger.warning(
-                "The 'persona_prompt' parameter is deprecated and will be removed in a future version of this function. Use 'preamble_override' instead.",
-            )
-        if persona_name is not None:
-            logger.warning(
-                "The 'persona_name' parameter is deprecated and will be removed in a future version of this function.",
-            )
-        if user_name is not None:
-            logger.warning(
-                "The 'user_name' parameter is deprecated and will be removed in a future version of this function.",
-            )
+        logger.warning(
+            "The 'reply' attribute is deprecated and will be removed in a future version of this function. Use 'text' instead.",
+        )
```

Comment:
shouldn't this be added inside a getter instead; so that the deprecation warning is not spamming the console, only when a user accesses `chat.reply`